docs/public-networks/reference/api/index.md

```diff
@@ -4850,9 +4850,8 @@ None
 | Network ID | Chain | Network | Description
 |------------|-------|---------|-------------------------------|
 | `1`        | ETH   | Mainnet | Main Ethereum network         |
-| `3`        | ETH   | Ropsten | PoS test network              |
-| `4`        | ETH   | Rinkeby | PoA test network using Clique |
-| `5`        | ETH   | Goerli  | PoA test network using Clique |
+| `5`        | ETH   | Goerli  | PoS test network              |
+| `11155111` | ETH   | Sepolia | PoS test network              |
```

Comment:
Since we removed Ropsten in the chart, should we update the example below this from Ropsten to Goerli or Sepolia? 

There's also a Ropsten example at [`eth_getTransactionByBlockNumberAndIndex`](https://hyperledger-besu--1129.org.readthedocs.build/en/1129/public-networks/reference/api/#eth_gettransactionbyblocknumberandindex) and the third point of the warning at the top of the page.  Should we update these?

Reply:
I figured to leave the mention of Ropsten in the mining page and the `eth_getTransactionByBlockNumberAndIndex` example, since those only reference specific historical uses of Ropsten. However, I added a note to the top of the mining page to indicate Ropsten is deprecated, and changed the example under the `net_version` example.